includes/discovery/libvirt-vminfo.inc.php

```diff
@@ -9,18 +9,23 @@
 
     $ssh_ok = 0;
 
+    $userHostname = $device['hostname'];
+    if (!is_null($config['libvirt_username'])) {
```

Comment:
That's true, as unlikely as a username of 0 might be ;D.
But I do think isset is more clear for readability and isset is only true if the variable is not null.